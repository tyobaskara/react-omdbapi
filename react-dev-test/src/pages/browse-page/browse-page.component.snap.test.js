import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import BrowsePage from './browse-page.component';

jest
  .mock('../../components/movies-overview/movies-overview.container', () => 'MoviesOverview')
  .mock('../movie-page/movie-page.container', () => 'MoviePage');

test('BrowsePage snaptest', () => {
  const props = {
    match: {
      path: '/'
    }
  };
  const component = renderer.create(
    <BrowserRouter>
      <BrowsePage {...props} />
    </BrowserRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});