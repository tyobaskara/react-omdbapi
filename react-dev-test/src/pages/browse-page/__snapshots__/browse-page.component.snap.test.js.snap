// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowsePage snaptest 1`] = `
<div
  className="browse-page"
>
  <MoviesOverview
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
    location={
      Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      }
    }
    match={
      Object {
        "isExact": true,
        "params": Object {},
        "path": "/",
        "url": "/",
      }
    }
  />
</div>
`;
